catalog/ui/catalog-ui-search/webpack/config/devServer.js

```diff
@@ -20,6 +20,7 @@ var handleProxyRes = function (proxyRes, req, res) {
 
 var proxyConfig = function (target) {
     return {
+        ws: true,
```

Comment:
I'm assuming `ws` is websocket?  If this isn't some convention, is it possible to use `webSocket` instead?

Reply:
This is the config option specified by webpack.